src/app/account-select/account-select.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="section--accounts" *ngIf="hasUsers">
-  <div class="section--accounts__header margin-bottom--medium">
+  <div class="section--accounts__header margin-bottom--small">
```

Comment:
something is looking funky here, can we get rid of the white border at the bottom? Also, maybe we want a slicker looking scrollbar: 
![image](https://github.com/mosster/identity/assets/81658138/f5c808e6-cdf1-411b-bfab-b63492aea81c)